src/app/page.tsx

```diff
@@ -1,11 +1,124 @@
-import HomePlaceholder from '@/components/Home/HomePlaceholder';
+import Link from 'next/link';
 import React from 'react';
+import Image from 'next/image';
+import { homePageData } from '../../config/home.config';
 
 function Home() {
+  const icons = Object.keys(homePageData.community)
+  const iconLinks = Object.values(homePageData.community)
+
   return (
-    <div className="flex h-screen flex-col items-center justify-center">
-      <HomePlaceholder />
-    </div>
+    <main className="flex h-screen flex-col gap-y-56 bg-[url(/assets/apply/apply-bg.png)] bg-cover pt-24 text-white">
+      <div className="flex flex-col gap-x-11 md:flex-col   lg:flex-row">
+        <div className="flex max-w-3xl flex-col gap-y-4 rounded-r-2xl border-y border-r border-primary/50 bg-gray-300/10 px-24 py-14">
+          <h1 className="text-4xl font-semibold md:text-5xl lg:text-7xl">
+            association of computing machinery
+          </h1>
+          <p className="text-lg lg:text-2xl">
+            We&apos;re a team of students with one goal: building a greater, more collaborative
+            computing community at The University of Texas at Dallas.
+          </p>
+          <div className="flex w-fit flex-row gap-x-6 bg-acm-gradient px-8 py-4">
+            {icons.map((icon: string, idx: number) => {
+              return (
+                <a href={iconLinks[idx]} key={idx}>
+                  <Image
+                    src={`/assets/home/icons/${icon}.png`}
+                    alt=""
+                    className="object-contain"
+                    width={25}
+                    height={25}
+                  />
+                </a>
+              );
+            })}
+          </div>
+        </div>
+        <div className="mt-12 flex flex-col gap-y-10">
+          <div className="flex max-w-sm flex-col items-start rounded-2xl border-primary/50 bg-gray-300/10 py-6 pl-28 pr-3">
+            <p className="text-xl">
+              join our mailing list to stay updated with our events and programs!
+            </p>
+            <div className="mt-2 self-end">
+              <Link
+                href="/"
+                className="flex h-[2.30rem] w-fit items-center justify-center bg-acm-gradient px-4 py-2 text-base font-bold md:text-xl"
+              >
+                apply
+              </Link>
+            </div>
+          </div>
+          <div className="flex max-w-sm flex-col items-start rounded-2xl border-primary/50 bg-gray-300/10 py-6 pl-28 pr-3">
+            <p className="text-xl">acm applications now open for fall 2023!</p>
+            <div className="mt-2 self-end">
+              <Link
+                href="/"
+                className="flex h-[2.30rem] w-fit items-center justify-center bg-acm-gradient px-4 py-2 text-base font-bold md:text-xl"
+              >
+                apply
+              </Link>
+            </div>
+          </div>
+        </div>
```

Comment:
it might be better to store the contents of these 2 blocks in `homePageData` in case we wanna change what goes there later on.